packages/Toggle/src/Toggle.js

```diff
@@ -1,7 +1,352 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import { ccClassName } from '@leafygreen-ui/lib';
-import * as style from './style';
+import { createDataProp, emotion } from '@leafygreen-ui/lib';
+import { colors } from '@leafygreen-ui/theme';
+
+const { css, cx } = emotion;
+
+const toggleInput = createDataProp('toggle-input');
+const toggleGroove = createDataProp('toggle-groove');
+
+const transitionInMS = 150;
+
+const inputStyle = css`
+  margin: 0;
+  position: absolute;
+  left: 100%;
+  top: 100%;
+  pointer-events: none;
+  opacity: 0;
+`;
+
+// We use a div for the focus state rather than a pseudo-element
+// because said pseudo-element would need to be on the label element
+// which can't use the contained input's focus pseudo-class.
+const focusStateStyle = css`
+  transition: all ${transitionInMS}ms ease-in-out;
+  position: absolute;
+  top: -2px;
+  bottom: -2px;
+  left: -2px;
+  right: -2px;
+  border: 2px solid rgba(67, 177, 229, 0.25);
+  border-radius: 50px;
+  opacity: 0;
+  transform: scale(0.8);
+
+  ${toggleInput.selector}:focus ~ & {
+    opacity: 1;
+    transform: scale(1);
+  }
+`;
+
+const labelStyleBase = `
+  transition: all ${transitionInMS}ms ease-in-out;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+  height: 11px;
+  line-height: 11px;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 11px;
+  color: white;
+  user-select: none;
+`;
+
+const onLabelStyle = css`
+  ${labelStyleBase};
```

Comment:
Why not just have this as part of `cx` on the components? Compose the styles on the element as opposed to making an 'extended' style.

Reply:
Yep, done!